aspnetcore/blazor/state-management.md

```diff
@@ -239,7 +230,7 @@ In any component that requires loading or saving data to browser storage, use th
 The choice depends on which browser storage location you wish to use. In the following example, `sessionStorage` is used:
 
 ```razor
-@using Microsoft.AspNetCore.Components.Web.Extensions
+@using Microsoft.AspNetCore.Componets.Server.ProtectedBrowserStorage
```

Comment:
```suggestion
@using Microsoft.AspNetCore.Components.Server.ProtectedBrowserStorage
```

Reply:
Should we retitle this section to say 

> ProtectedBrowserStorage in Blazor Server

Reply:
It's on a doc "pivot," so they'll select "Blazor Server" at the top and the entire topic will be on Blazor Server.